openedx/features/course_experience/.eslintrc.js

```diff
@@ -2,6 +2,10 @@ module.exports = {
   extends: 'eslint-config-edx',
   root: true,
   settings: {
-    'import/resolver': 'webpack',
+    'import/resolver': {
+      webpack: {
+        config: 'webpack.dev.config.js',
```

Comment:
course_experience was already es2015 and only had a couple eslint errors dealing with edx-ui-toolkit imports.  This fixes them.